### PR TITLE
My solution

### PR DESCRIPTION
**Pull request by kovaliskoveronika**
Admin page > cars
screen_admin_page_cars
Admin page > drivers
screen_admin_page_drivers
Admin page > manufacturers
screen_admin_page_manufacturers

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+venv
+__pycache__
+.idea
+db.sqlite3
```
Comment (YuriiKindrat) [inline .gitignore:4]: no blank line at the end of file (

File: taxi/admin.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    list_display = UserAdmin.list_display + ("license_number",)
+    fieldsets = UserAdmin.fieldsets + (
+        ("Additional info",
+         {"fields": ("license_number",)}
+         ),
+    )
+    add_fieldsets = UserAdmin.add_fieldsets + (
+        ("Additional info",
+         {"fields": ("license_number",)}
+         ),
```
Comment (YuriiKindrat) [inline taxi/admin.py:18]: bad formmating here (

File: taxi/models.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return f"name: {self.name}, country: {self.country}"
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255,
```
Comment (YuriiKindrat) [inline taxi/models.py:16]: same

File: taxi/models.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+                                      unique=True,
+                                      )
+
+    class Meta:
+        verbose_name = "Driver"
+        verbose_name_plural = "Drivers"
```
Comment (YuriiKindrat) [inline taxi/models.py:22]: did u read checklist ?

File: taxi/models.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(to=Manufacturer,
+                                     on_delete=models.CASCADE,
+                                     related_name="cars"
+                                     )
+    drivers = models.ManyToManyField(to=settings.AUTH_USER_MODEL,
+                                     related_name="cars"
+                                     )
```
Comment (YuriiKindrat) [inline taxi/models.py:33]: same about formmating

**Review by YuriiKindrat — CHANGES_REQUESTED** [review]

File: taxi_service/settings.py [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'taxi',
+]
```
Comment (Oleg-s-s) [inline taxi_service/settings.py:41]:
```suggestion
INSTALLED_APPS = [
    'django.contrib.admin',
    'django.contrib.auth',
    'django.contrib.contenttypes',
    'django.contrib.sessions',
    'django.contrib.messages',
    'django.contrib.staticfiles',
    
    'taxi',
]
```

**Review by Oleg-s-s — APPROVED** [review]
Its better to clean your project from unnecessary docstrings and comments

**Pull request closed without merging**